src/forms/Textarea.jsx

```diff
@@ -113,82 +117,47 @@ export class Textarea extends React.PureComponent<Props, State> {
 			name,
 			value, // eslint-disable-line no-unused-vars
 			label,
-			labelClassName,
-			className,
-			error,
+			labelClassName, // eslint-disable-line no-unused-vars
```

Comment:
I'd suggest removing labelClassName and any other unused props from the props type. Thinking back on this, during QA and during the intermediate phase of having MWC + Swarm UI, keeping the prop definitions up to date will help eliminate a lot of confusion.

Reply:
I disagree.. If we remove this from being destructured but unused here, then it would get passed all the way through (here: https://github.com/meetup/swarm-ui/blob/master/packages/swarm-components/src/Textarea.jsx#L32) and we'd get errors about passing down labelClassName to the dom element. So if we do remove this, we will have to remove the prop everywhere in mup-web where the prop (or any other now-unused prop) is passed.

I think we should leave these formerly used props, and perhaps add a console warning about it.

Reply:
@jaclynj I had the same thoughts. @sadafie means to clean up the proptype definitions on line 18, but she wasn't able to comment on that line.

Reply:
ah got it